tests/lib/rules/keyword-spacing.js

```diff
@@ -545,6 +545,7 @@ ruleTester.run("keyword-spacing", rule, {
         { code: "if (a) {}else{}", options: [override("else", NEITHER)] },
         "if (a) {}\nelse\n{}",
         { code: "if(a) {}\nelse\n{}", options: [NEITHER] },
+        { code: "if(a){ }else{ }", options: [{ before: false, after: true, overrides: { else: { after: false }, if: { after: false } } }] },
```

Comment:
Could we please also add a test to show that a missing `after` in `overrides` will not override the common `after`?

Reply:
@mdjermanovic 
I did it, Should I add more test cases?

Reply:
Thanks! I think that these two tests cover this situation well.